Problems with WebView

Hi. 

I get the following error when compiling

```
app/build/generated/source/view-inspector/beta/debug/app/view_inspector/ProbeProxy$android_widget_WebView.java
Error:(8, 22) error: cannot find symbol class WebView
Error:(9, 62) error: cannot find symbol class WebView
Error:(8, 22) error: cannot find symbol class WebView
Error:(9, 62) error: cannot find symbol class WebView
Error:(20, 19) error: method onMeasure in class Interceptor cannot be applied to given types;
required: View,int,int
found: ProbeProxy$android_widget_WebView,int,int
reason: actual argument ProbeProxy$android_widget_WebView cannot be converted to View by method invocation conversion
```

--- a/view-inspector-plugin/src/main/groovy/view_inspector/plugin/internal/LayoutResourceParser.groovy
+++ b/view-inspector-plugin/src/main/groovy/view_inspector/plugin/internal/LayoutResourceParser.groovy
@@ -54,6 +54,8 @@ class LayoutResourceParser {
     def className
     if (name == 'View') {
       className = "android.view.View"
+    } else if (name == 'WebView') {
+      className = "android.webkit.WebView"
     } else if (
         name == "merge" ||
         name == "include" ||

